.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "azure-mock/data"]
+	path = azure-mock/data
+	url = git@github.com:Hochfrequenz/machine-readable_anwendungshandbuecher.git
```

macht das nicht mit gitkraken probleme?
```suggestion
	url = https://github.com/Hochfrequenz/machine-readable_anwendungshandbuecher
```

---

nope worked for me ^^

---

Ich habe bei mir aber auch einen ssh-agent konfiguriert.
Kannst du mal schauen ob das bei klappt?